src/doc/rustdoc/src/unstable-features.md

```diff
@@ -467,3 +467,20 @@ $ rustdoc src/lib.rs -Z unstable-options --runtool valgrind
 ```
 
 Another use case would be to run a test inside an emulator, or through a Virtual Machine.
+
+### `--source-code-external-url`: using external source code
+
+In case you don't to generate the source code files and instead use existing ones (available
+through an HTTP URL!), you can use this option:
```

Comment:
```suggestion
through a HTTP URL!), you can use this option:
```

Reply:
I actually think `an` is right here, because the H would be pronounced with a leading a sound.

Reply:
So looking into this, it's surprisingly not definitive. The best answer I could find is that if it starts with a vowel sound, then it's "an", e.g. "an FAQ" (pronounced letter by letter), and if it starts with a consonant then it should be "a". So by this method, it would be "A HTTP" since it's pronounced ("Haitch"-"Tee"-"Tee"-"Pee). Which is what sounds right in my head. It's an incredibly minor nit though.

https://english.stackexchange.com/questions/1016/do-you-use-a-or-an-before-acronyms-initialisms#1017

Reply:
I think it also depends on your accent, 'Haitch' to me sounds British/European and everyone I know pronounces it 'Aitch'. So yeah, not definitive.

Reply:
Well I am European. :) I believe our policy has been an American-English for English.